lib/mox.ex

```diff
@@ -538,7 +538,30 @@ defmodule Mox do
   def expect(mock, name, n \\ 1, code)
       when is_atom(mock) and is_atom(name) and is_integer(n) and n >= 0 and is_function(code) do
     calls = List.duplicate(code, n)
-    add_expectation!(mock, name, code, {n, calls, nil})
+    arity = arity(code)
+    add_expectation!(mock, name, arity, {n, calls, nil})
+    mock
+  end
+
+  @doc """
+  Ensures that `name` in `mock` with arity `arity` is not invoked.
+
+  When `reject/3` is invoked, any previously declared `stub` for the same `name` and arity will
+  be removed. This ensures that `reject` will fail if the function is called. If a `stub/3` is
+  invoked **after** `reject/3` for the same `name` and arity, the stub will be used instead, so
+  `reject` will have no effect.
+
+  ## Examples
+
+  To expect `MockWeatherAPI.get_temp/1` to never be called:
+
+      reject(MockWeatherAPI, :get_temp, 1)
+  """
+  @doc since: "1.2.0"
```

Comment:
Guessing next version number here (not sure if this or `1.1.1`)

Reply:
It's this per SemVer 👍